src/schema/schema.graphql

```diff
@@ -1320,12 +1321,38 @@ type SignatureResponse {
 	signature: [NameId]
 }
 
+type Document {
+	id: ID
+	folderId: ID
+	name: String
+	version: Float #same item may have different versions (i.e same names) will need to implement ListDocumentRevisionsRequest 
+	contentType: String
+ 	descEnabled: Boolean
+	date: Float
+	changeDate: Float
+	modifiedSequence: Float
+	revision: Float
+	size: Float
+	sortField: String
+	tags: String # t
```

Comment:
we should put shorterForm of keys in comments, just like this for all full versioned keys

Reply:
Done

Reply:
Done